distribution/lib/Standard/Base/0.0.0-dev/src/Errors/Common.enso

```diff
@@ -476,3 +477,19 @@ type Additional_Warnings
     to_display_text : Text
     to_display_text self =
         "There were "+self.count.to_text+" additional issues."
+
+type Loss_Of_Numeric_Precision
```

Comment:
Do we want to merge `Loss_Of_Integer_Precision` with this error?

Reply:
`Loss_Of_Integer_Precision` has a more detailed explanation. I don't want this explanation to be lost by merge.

If we merge, we should ensure that the error messages clearly explain what is the problem.